modules/nw-ovn-kubernetes-matrix.adoc

```diff
@@ -24,7 +24,11 @@ ifeval::["{context}" == "about-ovn-kubernetes"]
 
 |IPsec encryption for intra-cluster communication|Supported|Not supported
 
-|IPv6|Supported ^[3]^ ^[4]^|Not supported
+|IPv4-primary dual-stack|Supported|Supported ^[3]^
+
+|IPv6 single-stack networking|Supported ^[4]^|Unsupported
+
+|IPv6 IPv6-primary dual-stack networking|Supported ^[5]^|Unsupported
```

Comment:
To be consistent with 'IPv4-primary dual-stack' above, this can be 'IPv6-primary dual-stack'

Reply:
:heavy_check_mark: